helpers/migration.ts

```diff
@@ -44,6 +47,27 @@ class ObjectSchemaHelper<T extends ObjectSchemaDefinition<string, string>> {
   }
 }
 
+export function paramDefHelper<S extends UnionType, T extends ParamDef<S>>(def: T): ParamDefHelper<S, T> {
+  return new ParamDefHelper(def);
+}
+
+class ParamDefHelper<S extends UnionType, T extends ParamDef<S>> {
+  private readonly _def: T;
+
+  constructor(def: T) {
+    this._def = def;
+  }
+
+  get defaultValue(): SuggestedValueType<S> | undefined {
+    return this._def.suggestedValue ?? this._def.defaultValue;
+  }
+
+  get suggestedValue() {
```

Comment:
Nit, the approach I was taking with the other helpers was just to have one way of referring to the value, i.e. eliminate this redundant getter. Then the changes in code that references these fields is simply wrapping the parameter access with the helper, leaving the field name unchanged, hopefully making it easy to review the changes for correctness and safety.